apis/config/types_landscaper_config.go

```diff
@@ -25,11 +25,28 @@ type LandscaperConfiguration struct {
 	// CrdManagement configures whether the landscaper controller should deploy the CRDs it needs into the cluster
 	// +optional
 	CrdManagement *CrdManagementConfiguration `json:"crdManagement,omitempty"`
-	// DeployItemPickupTimeout defines how long a deployer can take to react on changes to a deploy item before the landscaper will mark it as failed.
+	// DeployItemTimeouts contains configuration for multiple deploy item timeouts
+	// +optional
+	DeployItemTimeouts *DeployItemTimeouts `json:"deployItemTimeouts,omitempty"`
+}
+
+// DeployItemTimeouts contains multiple timeout configurations for deploy items
+type DeployItemTimeouts struct {
+	// PickupTimeout defines how long a deployer can take to react on changes to a deploy item before the landscaper will mark it as failed.
 	// Allowed values are 'none' (to disable pickup timeout detection) and anything that is understood by golang's time.ParseDuration method.
 	// Defaults to five minutes if not specified.
 	// +optional
-	DeployItemPickupTimeout string `json:"deployItemPickupTimeout,omitempty"`
+	Pickup string `json:"pickup,omitempty"`
+	// Abort specifies how long the deployer may take to abort handling a deploy item after getting the abort annotation.
+	// Allowed values are 'none' (to disable abort timeout detection) and anything that is understood by golang's time.ParseDuration method.
+	// Defaults to five minutes if not specified.
+	// +optional
+	Abort string `json:"abort,omitempty"`
```

Comment:
as i currently see the string the copied documentation of allowed values.
I would propose to do something like kubernetes did with the `metav1.Time`.
So basically lets create our own duration wrapper that implements the json marshal/unmarshal methods + the opeapispec values.
With that we can simply write:
```
Abort *Duration `json:"abort,omitempty"`
```
So we do not need special parse logic in the business logic and the api seems clearer as we actually want a duration rather than a string.